corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -39,10 +39,15 @@ def setUpClass(cls):
             ReportAppConfig(report_id='master_report_id', header={'en': 'CommBugz'}),
         ]
 
-        cls.plain_master_app = Application.new_app(cls.domain, "Master Application")
+        cls.master1 = Application.new_app(cls.domain, "First Master Application")
         cls.linked_domain = 'domain-2'
-        cls.plain_master_app.linked_whitelist = [cls.linked_domain]
-        cls.plain_master_app.save()
+        cls.master1.linked_whitelist = [cls.linked_domain]
+        cls.master1.save()
+
+        cls.master2 = Application.new_app(cls.domain, "Second Master Application")
+        cls.linked_domain = 'domain-2'
+        cls.master2.linked_whitelist = [cls.linked_domain]
+        cls.master2.save()
```

Comment:
Does this test change imply that we're expecting all linked apps to have access to multi-master? If not, would it be useful to have some explicit single-master tests around?

Reply:
Expecting all linked apps to have access to multi-master.